internal/api/connectors/inventory/inventory_test.go

```diff
@@ -102,6 +105,7 @@ var _ = Describe("Inventory", func() {
 			Expect(*resultData.SatelliteInstanceID).To(Equal("5678"))
 			Expect(*resultData.SatelliteVersion).To(Equal("6.11.3"))
 			Expect(*resultData.SatelliteOrgID).To(Equal("5"))
+			Expect(*resultData.AnsibleHost).To(Equal("host1.mydomain.com"))
```

Comment:
Like the other test failure.  This test is going to fail because the resultData.AnsibleHost field will be nil. In order to resolve this, you will need to add the ansible_host field to the mocked data in line 89.